File: C/Ex. 19.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    float a, t, v0, v, vf ;

    printf ("Digite o valor da aceleracao: ");
    scanf ("%f", & a);

    printf ("Digite o tempo levado no percurso: ");
    scanf ("%f", & t);

    printf ("Digite a velocidade inicial: ");
    scanf ("%f", & v0);

    v=v0+a*t;

    vf=v*3.6;

    if (vf<=40)
    {
        printf ("Veiculo Muito lento");
    }

    else 
    {
        if (vf>40 && vf<=60)
        {
            printf ("Velocidade permitida");
        }

        else 
        {
            if (vf>60 && vf<=80)
            {
                printf ("Velocidade de cruzeiro");
            }

            else 
            {
                if (vf>80 && vf<=120)
                {
                    printf ("Veiculo rapido");
                }

                else 
                {
                    printf ("Veiculo muito rapido");
                }
            }
        }        
    }
}

File: Python/Ex 19.py
a = float(input("Digite o valor da aceleracao: "))
t = float(input("digite qual o tempo levado no percurso: "))
v0 = float(input("Digite o valor da velocidade inicial: "))

v=v0+a*t

vf=v*3.6

if (vf<=40):
    print("Veiculo Muito lento")
elif ((vf>40) and (vf<=60)):
    print("velocidade permitida")
elif ((vf>60) and (vf<=80)):
    print("velocidade de cruzeiro")
elif ((vf>80) and (vf<=120)):
    print("veiculo rapido")
else:
    print("Veiculo muito rapido")
